guides/doc-Administering_Project/master.adoc

```diff
@@ -12,6 +12,16 @@ include::common/assembly_accessing-server.adoc[leveloffset=+1]
 
 include::common/modules/proc_starting-and-stopping-server.adoc[leveloffset=+1]
 
+ifdef::satellite[]
+// Cloning Satellite Server
```

Comment:
```suggestion
```

If you feel like the file name "cloning-satellite-server" is not descriptive yet, then add what's missing as a comment. replicating the file name does IMHO not help all that much.

I'd remove this comment but focus on a great commit message explaining _why_ we move the content from chapter A to B.

Reply:
@maximiliankolb I see your point. I kept the comment sine it was part of the conditional in the Upgrade guide.